docs/content/en/_index.md

```diff
@@ -46,11 +46,11 @@ cascade:
 </div>
 <div class="usecasebox">
 
-## Use Cases
+## Use cases
 
-We extend the K8s APIs with the following Use Cases:
+We extend the K8s APIs with the following use cases:
 
-<div class="row usecases">
+<div class="row use cases">
```

Comment:
pls don't change that, it's needed in html to be one word. otherwise you would need to also change the CSS referencing the class here

Reply:
Ah, okay.  "usecases" was flagged by the spell checker although "usecase" seems to be okay.  Actually, the prose should work just fine if it is "use cases" - both should be acceptable words, right?